activerecord/lib/active_record/railtie.rb

```diff
@@ -38,6 +38,7 @@ class Railtie < Rails::Railtie # :nodoc:
     config.active_record.raise_on_assign_to_attr_readonly = false
     config.active_record.belongs_to_required_validates_foreign_key = true
     config.active_record.generate_secure_token_on = :create
+    config.active_record.default_timezone = :utc
```

Comment:
Should we follow the app-wide configuration by default?

Reply:
I am not sure I understood the question. This setting is only app wide. It should not be confused with 'Time.zone' and 'config.time_zone'.

Reply:
Sorry for not being clear. My question was "should we default to `config.time_zone`?"

Reply:
This setting unlike 'config.zone' can only take values 'local' and 'utc'.